src/problems.jl

```diff
@@ -31,6 +30,8 @@ function Base.getproperty(p::Problem, s::Symbol)
         else
             return objective_value(p)
         end
+    elseif s === :size
+        return (1, 1)
```

Comment:
I'm not so sure what this is for as well. It actually wasn't tested so I had to add a test

Reply:
hm, if it was coming up in codecov as untested maybe it's not used at all and we can delete it. I don't really remember this, but if I had to guess it would be for something like partially specified problems (#310)

Reply:
Let's try to remove it and see if anything breaks:
https://github.com/jump-dev/Convex.jl/pull/504/commits/5b401f0d9615f173a34d93c2ba2f3080f210c003